start-server-and-test.js

```diff
@@ -0,0 +1,102 @@
+const { spawn } = require('child_process');
+const fetch = require('node-fetch');
+
+/* eslint-disable no-console */
+
+const serverUrl = 'http://localhost:4000';
+const serverTimeout = 360000; // 6 minutes
```

Comment:
This means that the max time the visual test could take is 6 minutes right? Why 6 minutes?

Reply:
nvm just noticed that this was the sever start timeout right?

Reply:
Yeah, that's the server timeout. I just assumed the worst case time as I was tired with the constant timeouts while testing 😅